src/angular/components/sso.component.ts

```diff
@@ -35,6 +35,17 @@ export class SsoComponent {
     protected redirectUri: string;
     protected state: string;
     protected codeChallenge: string;
+    protected returnUri: string;
+    protected linkLoggedInUser: boolean;
```

Comment:
Can this just be made a parameter of `buildAuthorizeUrl` instead? I am also thinking that `buildAuthorizeUrl` should probably be protected rather than public since it doesnt need to be called from the UI.

Reply:
Done, but for context this also involved adding some optional params to submit()